en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -91,7 +91,7 @@ SS_BASE_URL="http://localhost/"
 ```
 
 You will also need to be assigned the following permissions. Contact one of the SilverStripe staff from
-the [core committers](core_committers), who will assist with setting up your credentials.
+the [core committers](/project_governance/core_committers), who will assist with setting up your credentials.
```

Comment:
There's another one of these at the bottom of the page to update.

Reply:
Fixed that link and a few minor unrelated links.